atlasdb-client/src/main/java/com/palantir/atlasdb/persistent/rocksdb/RocksDbPersistentStore.java

```diff
@@ -46,52 +46,44 @@
 import com.google.common.collect.Maps;
 import com.google.common.collect.Streams;
 import com.palantir.atlasdb.persistent.api.ImmutableStoreNamespace;
-import com.palantir.atlasdb.persistent.api.PersistentTimestampStore;
-import com.palantir.atlasdb.table.description.ValueType;
+import com.palantir.atlasdb.persistent.api.PersistentStore;
 import com.palantir.common.streams.KeyedStream;
 import com.palantir.logsafe.Preconditions;
 import com.palantir.tracing.Tracers.ThrowingCallable;
 
 /**
- * Implementation of the {@link PersistentTimestampStore} using RocksDB as the underlying persistent storage. Commit
- * timestamp associated with the start timestamp is encoded using delta encoding. Created {@link StoreNamespace}s are
- * backed by RocksDB ColumnFamilies such that calling {@link RocksDbPersistentTimestampStore#createNamespace(String)}
- * with the same name will construct a new {@link ColumnFamilyHandle} for each call.
+ * Implementation of the {@link PersistentStore} using RocksDB as the underlying persistent storage. Created
+ * {@link StoreNamespace}s are backed by RocksDB ColumnFamilies such that calling
+ * {@link RocksDbPersistentStore#createNamespace(String)} with the same name will construct a new
+ * {@link ColumnFamilyHandle} for each call.
  */
-public final class RocksDbPersistentTimestampStore implements PersistentTimestampStore {
-    private static final Logger log = LoggerFactory.getLogger(RocksDbPersistentTimestampStore.class);
+public final class RocksDbPersistentStore implements PersistentStore {
```

Comment:
It's still unclear to me the purpose of the refactoring here, this can still be in terms of timestamps no? do we actually have another use case for rocksdb?